Exercise_1.py

```diff
@@ -1,18 +1,29 @@
+Time complexity: O(1) for isEmpty, push, pop, peek. O(n) for size and show.
+Space complexity: O(1) for all
```

Comment:
Just to clarify, space complexity for each operation here is O(1) but the overall space is O(n) which is used by the stack.